crates/aquascope/src/interpreter/step.rs

```diff
@@ -533,9 +540,10 @@ impl<'mir, 'tcx> VisEvaluator<'mir, 'tcx> {
             Either::Left(_mplace) => {
               // todo!()
             }
-            Either::Right((frame, local, _)) => {
+            // FIXME(gavinleroy) is the `frame` now the third slot?
```

Comment:
ignore the `FIXME` but here is where you see the change that frames are no longer kept in the place local struct.